app/frontend/src/components/forms/submission/StatusPanel.vue

```diff
@@ -235,59 +236,77 @@ export default {
             throw new Error(this.$t('trans.statusPanel.status'));
           }
 
-          const statusBody = {
+          const baseStatusBody = {
             code: this.statusToSet,
-            submissionUserEmail: this.submissionUserEmail,
             revisionNotificationEmailContent: this.emailComment,
           };
-          if (this.showAssignee) {
-            if (this.assignee) {
-              statusBody.assignedToUserId = this.assignee.userId;
-              statusBody.assignmentNotificationEmail = this.assignee.email;
-            }
-          }
-          const statusResponse = await formService.updateSubmissionStatus(
-            this.submissionId,
-            statusBody
-          );
-          if (!statusResponse.data) {
-            throw new Error(
-              this.$t('trans.statusPanel.updtSubmissionsStatusErr')
-            );
+
+          if (this.showAssignee && this.assignee) {
+            baseStatusBody.assignedToUserId = this.assignee.userId;
+            baseStatusBody.assignmentNotificationEmail = this.assignee.email;
           }
 
-          if (this.emailComment) {
-            let formattedComment;
-            if (this.statusToSet === 'ASSIGNED') {
-              formattedComment = `Email to ${this.assignee.email}: ${this.emailComment}`;
-            } else if (
-              this.statusToSet === 'REVISING' ||
-              this.statusToSet === 'COMPLETED'
-            ) {
-              formattedComment = `Email to ${this.submissionUserEmail}: ${this.emailComment}`;
-            }
+          if (this.statusToSet === 'REVISING') {
+            // Handle multiple emails for REVISING
+            for (const user of this.selectedUsers) {
+              const statusBody = {
+                ...baseStatusBody,
+                submissionUserEmail: user.email,
+              };
+              const statusResponse = await formService.updateSubmissionStatus(
+                this.submissionId,
+                statusBody
+              );
 
-            const submissionStatusId =
-              statusResponse.data[0].submissionStatusId;
-            const user = await rbacService.getCurrentUser();
-            const noteBody = {
-              submissionId: this.submissionId,
-              submissionStatusId: submissionStatusId,
-              note: formattedComment,
-              userId: user.data.id,
+              if (!statusResponse.data) {
+                throw new Error(
+                  i18n.t('trans.statusPanel.updtSubmissionsStatusErr')
```

Comment:
Just need to import i18n from '~/internationalization';

Reply:
I've resolved the conflicts after your merge so this is no longer being used.